### PR TITLE
[NFC] Test behavior of common global variables

### PR DESCRIPTION
**Pull request by LU-JOHN**
Add tests to ensure that:

1. A common global without addrspace is reported as an error
2. A common addrspace global is not transformed to a locally allocated variable

Updated error message from "can not" to "cannot".  Otherwise this is a NFC.

### CODE REVIEW

**Review by vmaksimo — APPROVED** [review]

File: test/CommonAddrspaceGlobalVar.ll [hunk 1]
```diff
@@ -0,0 +1,20 @@
+; Ensure that a "common global" is not converted to a locally allocated
```
Comment (MrSidims) [inline test/CommonAddrspaceGlobalVar.ll:1]: test/CommonAddrspaceGlobalVar.ll -> test/transcoding/CommonAddrspaceGlobalVar.ll

File: test/CommonGlobalVarNoAddrspace.ll [hunk 2]
```diff
@@ -0,0 +1,24 @@
+; Ensure "common global" with no addrspace is reported as an error
```
Comment (MrSidims) [inline test/CommonGlobalVarNoAddrspace.ll:1]: quite unused directory, but should be placed here IMHO

test/CommonGlobalVarNoAddrspace.ll -> test/negative/CommonGlobalVarNoAddrspace.ll

File: test/CommonAddrspaceGlobalVar.ll (hunk 1 above)
Comment (LU-JOHN) [reply test/CommonAddrspaceGlobalVar.ll]: done

File: test/CommonGlobalVarNoAddrspace.ll (hunk 2 above)
Comment (LU-JOHN) [reply test/CommonGlobalVarNoAddrspace.ll]: done

**Review by MrSidims — APPROVED** [review]
test/GlobalVarNoAddrspace.ll - ideally should also be moved to negative, but it wasn't added in this patch, so I don't mind

**Comment by LU-JOHN** [comment]
> test/GlobalVarNoAddrspace.ll - ideally should also be moved to negative, but it wasn't added in this patch, so I don't mind

I've moved test/GlobalVarNoAddrspace.ll.     This is more consistent.

**Pull request merged**